src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className='ButtonPanel'>
+    <div className='group'>
+      <Button name='AC' />
+      <Button name='+/-' />
+      <Button name='%' />
+      <Button name='/' />
+    </div>
+    <div className='group'>
+      <Button name='7' />
+      <Button name='8' />
+      <Button name='9' />
+      <Button name='X' />
+    </div>
+    <div className='group'>
+      <Button name='4' />
+      <Button name='5' />
+      <Button name='6' />
+      <Button name='-' />
+    </div>
+    <div className='group'>
+      <Button name='1' />
+      <Button name='2' />
+      <Button name='3' />
+      <Button name='+' />
+    </div>
+    <div className='group'>
+      <Button name='0' />
+      <Button name='.' />
+      <Button name='=' />
+    </div>
```

Comment:
[Optional]
You can refactor this code a little bit more, think about an array containing arrays with the button names. Then you can iterate through it.